File: WYNK.Data/Model/ViewModel/BusinessRuleViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;


namespace WYNK.Data.Model.ViewModel
{
    public class BusinessRuleViewModel
    {

        public BusinessRule BusinessRule { get; set; }
        public BusinessRuleTran BusinessRuleTran { get; set; }
        public ICollection<BusinessRT> BusinessRT { get; set; }
        public ICollection<BusinessRT1> BusinessRT1 { get; set; }
        public string ModuleDescription { get; set; }
      
        
        public ICollection<IcdSPYarray> IcdSPYarray { get; set; }
        public ICollection<icdspyModuleDescription> icdspyModuleDescription { get; set; }

    }
    public class icdspyModuleDescription
    {
        public string ICDCODE { get; set; }
        

    }
    public class IcdSPYarray
    {
        public string ICDSPYDescription { get; set; }
        public int id { get; set; }
       
    }
    public class BusinessRT
    {
        public int BRID { get; set; }
        public int From { get; set; }
        public int TO { get; set; }
        public decimal Amount { get; set; }
        public int NODays { get; set; }
        public int CreatedBy { get; set; }
        public int? UpdatedBy { get; set; }
        public bool IsActive { get; set; }
        public DateTime? EffectiveDate { get; set; }
    }


    public class BusinessRT1
    {
        public DateTime? WEF { get; set; }
        public int BRID { get; set; }
        public int? From { get; set; }
        public int? TO { get; set; }
        public decimal Amount { get; set; }
        public int NODays { get; set; }
        public int CountOfRegTran { get; set; }

    }
}

File: WYNKWeb/src/app/Models/ViewModels/BusinessRuleViewModel.ts
import { DateTime } from 'wijmo/wijmo';
import { BusinessRule } from "../BusinessRule";
import { BusinessRuleTran } from "../BusinessRuleTran";


export class BusinessRuleD {
  BusinessRule: BusinessRule = new BusinessRule();
  BusinessRT: Array<BusinessRT> = [];
  BusinessRT1: Array<BusinessRT1> = [];
  IcdSPYarray: Array<IcdSPYarray> = [];
  icdspyModuleDescription: Array<icdspyModuleDescription> = [];
}
//export class BusinessR {
//  ModuleID: number;
//  ModuleDescription: string;
//  BRBasedOn: string;
//  CMPID: number;
//  IsActive: boolean;
//  ICDCODE: string;
//  IsAllSurgery: boolean;
//  IsTransacted: boolean;
//  WEF: Date;

//}
export class icdspyModuleDescription {

  ////////////////////BusinessRT1//////////////////////
  ICDCODE: string;
  


}
export class IcdSPYarray {

  ////////////////////BusinessRT1//////////////////////
  ICDSPYDescription: string;
  id: number;


}
export class BusinessRT1 {

  ////////////////////BusinessRT1//////////////////////
  BRID: number;
  From: number;
  TO: number;
  Amount: number;
  NODays: number;
  WEF: DateTime;

}
export class BusinessRT
{

  ////////////////////BusinessRT//////////////////////
  BRID: number;
  From: number;
  TO: number;
  Amount: number;
  NODays: number;

}


